<ui version="4.0" >
 <class>DlgPrefMidiDlg</class>
 <widget class="QWidget" name="DlgPrefMidiDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form2</string>
  </property>
  <widget class="Q3GroupBox" name="groupPowerMates" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>320</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Griffin PowerMates</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>31</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabelPowerMate1" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Device 1</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="ComboBoxPowerMate1" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="ComboBoxPowerMate2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="TextLabelPowerMate2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Device 2</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Q3GroupBox" name="groupBoxMice" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>100</y>
     <width>581</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title" >
    <string>Mice</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>561</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="TextLabel5_2_3_3_3" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Function</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel5_2_3_3" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Device</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="pushButtonMouseCalibrate1" >
       <property name="text" >
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="TextLabel5_2_3_3_3_2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Function</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="pushButtonMouseCalibrate2" >
       <property name="text" >
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" >
      <widget class="QPushButton" name="pushButtonMouseHelp" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="ComboBoxMouseDevice2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="TextLabel5_2_3_3_2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Device</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QComboBox" name="ComboBoxMouseFunction2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QComboBox" name="ComboBoxMouseFunction1" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="ComboBoxMouseDevice1" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Q3GroupBox" name="groupBoxMidi" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>82</height>
    </rect>
   </property>
   <property name="title" >
    <string>MIDI</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>16</y>
      <width>391</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel6" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Controller Mapping</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="TextLabel5" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Device</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="ComboBoxMidiconf" >
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="ComboBoxMididevice" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Q3GroupBox" name="groupHercules" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>320</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Hercules DJ Console</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabelPowerMate1_2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="text" >
        <string>Mapping</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="ComboBoxHercules" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
